common/app/views/fragments/atoms/snippets/snippetImage.scala.html

```diff
@@ -0,0 +1,25 @@
+@import layout.ContentWidths.MainMedia
```

Comment:
Why can't we use https://github.com/guardian/frontend/blob/d7a5d4b1647848ff35e9566cc8c6eb0b9c433e34/common/app/views/fragments/image.scala.html ?